sdk/cosmosdb/cosmos/src/request/request.ts

```diff
@@ -127,6 +127,11 @@ export async function getHeaders({
     headers[Constants.HttpHeaders.ConsistencyLevel] = options.consistencyLevel;
   }
 
+  if (options.maxIntegratedCacheStaleness) {
+    headers[Constants.HttpHeaders.DedicatedGatewayPerRequestCacheStaleness] =
```

Comment:
nit: is line splitting for limiting the width? 
Just observation all others have them in single line E2E.

Reply:
We have an auto formatter on all saves and checkins...I don't have control over that.